assets/src/dashboard/components/popoverPanel/index.js

```diff
@@ -14,27 +14,37 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * External dependencies
  */
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-/**
- * WordPress dependencies
- */
-import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import { ReactComponent as CloseIcon } from '../../icons/close.svg';
 import { Z_INDEX } from '../../constants';
-import { Pill } from '../pill';
+import { PILL_LABEL_TYPES } from '../../constants/components';
+import { ReactComponent as CloseIcon } from '../../icons/close.svg';
 import { DROPDOWN_ITEM_PROP_TYPE } from '../types';
-import { ColorDot } from '../colorDot';
+import Pill from '../pill';
+
+const visuallyHiddenStyles = {
+  position: 'absolute',
+  height: '1px',
+  width: '1px',
+  overflow: 'hidden',
+  clip: 'rect(1px, 1px, 1px, 1px)',
+  'white-space': 'nowrap',
+};
```

Comment:
I know this is just moving this up, but instead of an object if this is used inside a styled component we could use the `css` helper from `styled-components` instead.

Reply:
good idea!